components/textarea/stories/textarea.stories.js

```diff
@@ -0,0 +1,276 @@
+import { Sizes } from "@spectrum-css/preview/decorators";
```

Comment:
For now, let's leave these assets in the textfield component until we have a plan for migrating.